r2ai/interpreter.py

```diff
@@ -713,8 +711,7 @@ def keywords_ai(self, text):
         msg = f"Considering the sentence \"{text}\" as input, Take the KEYWORDS or combination of TWO words from the given text and respond ONLY a comma separated list of the most relevant words. DO NOT introduce your response, ONLY show the words"
         msg = f"Take \"{text}\" as input, and extract the keywords and combination of keywords to make a search online, the output must be a comma separated list" #Take the KEYWORDS or combination of TWO words from the given text and respond ONLY a comma separated list of the most relevant words. DO NOT introduce your response, ONLY show the words"
         response = mm(msg, stream=False, temperature=0.1, stop="</s>", max_tokens=1750)
-        if self.env["debug"] == "true":
-            print("KWSPLITRESPONSE", response)
+        LOGGER.debug("KWSPLITRESPONSE", response)
```

Comment:
use meaningful text here

Reply:
Hmm, that was there already xD

Should be network split response I guess?

Reply:
On probably just response?

Reply:
Or just logger.debug(response)